x-pack/plugins/observability/server/routes/slo/route.ts

```diff
@@ -5,15 +5,15 @@
  * 2.0.
  */
 
-import { createSloParamsSchema } from '../../schema';
+import { createSLOParamsSchema } from '../../types/schema';
 import { createObservabilityServerRoute } from '../create_observability_server_route';
 
 const createSLORoute = createObservabilityServerRoute({
   endpoint: 'POST /api/observability/slos',
   options: {
     tags: [],
   },
-  params: createSloParamsSchema,
+  params: createSLOParamsSchema,
   handler: async ({ context, request, params }) => {
     return { success: true };
```

Comment:
The following PR will introduce the service responsible for creating an SLO, using the services done in the other PR: resourceInstaller and repository